Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'sinatra', '< 2' if RUBY_VERSION < '2.2'
```

Comment:
Well it's a runtime dependency so needs to stay in the gemspec, but built gemspecs can't contain conditionals (such as `if RUBY_VERSION` etc.), so I think it'll have to be here too.